### ISSUE TITLE
react/prop-types said missing in props validation while use stateless component.

### ISSUE THREAD

**Issue by MicroBenz** [yannickcr/eslint-plugin-react#1016]
I've wrote stateless component and props validation like this.

```jsx
const BusDistance = ({ line }) => (
    <div className="box">
        <article className="media">
            <div className="media-left">
                <div style={styles.circleBusLine} className="red-bg">
                    <h1>{line}</h1>
                </div>
            </div>
            <div className="media-content">
                <h1>2 KM</h1>
            </div>
        </article>
    </div>
);

BusDistance.PropTypes = {
    line: PropTypes.number.isRequired,
};
```

And ESLint said.

![screen shot 2560-01-10 at 2 50 22 pm](https://cloud.githubusercontent.com/assets/9103800/21797885/3c70b288-d744-11e6-8a09-6e675e3b9735.png)

Please clarify me how to make a props validation with stateless component.

Thank you.

**Comment by hugihlynsson** [comment]
Glancing over this quickly, it seems that `BusDistance.PropTypes` should be `BusDistance.propTypes`, that is – propTypes must start with a lowercase p. This should resolve your problem.

**Comment by MicroBenz** [comment]
@hugihlynsson Problem Solved! Thank you :)
